migtests/tests/analyze-schema/expected_issues.json

```diff
@@ -217,9 +217,17 @@
         },
         {
             "ObjectType": "TABLE",
-            "ObjectName": "newtable",
-            "Reason": "Stored generated column not supported. Column is: id",
-            "SqlStatement": "CREATE TABLE newtable (\n\tid UUID GENERATED ALWAYS AS gen_random_uuid() STORED,\n\torg uuid NOT NULL,\n\tname text,\n\tcreated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP UNIQUE(name, org)\n);",
+            "ObjectName": "public.employees4",
+            "Reason": "Stored generated column not supported. Column is: first_name",
```

Comment:
+1

Reply:
Yes. Thanks for catching!

Reply:
Had a talk with @priyanshi-yb. Removed the column field for now. The regex is sufficient to capture the required DDL and we will require advanced handling to figure out the columns (since there can be multiple columns of this type.)
We can add it back in the future if the need arises or we switch to the parser.